.github/workflows/check-pkg-availability.yml

```diff
@@ -0,0 +1,129 @@
+                dnf download --resolve -y centreon-${{ inputs.module }}
+              else
+                dnf download --resolve -y centreon
+                dnf download --resolve -y centreon-${{ inputs.module }}
+              fi
+            ;;
+            almalinux:9:23.10 | almalinux:9:23.04)
+              dnf install -y dnf-plugins-core
+              dnf install -y epel-release
+              dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+              dnf config-manager --set-enabled crb
+              dnf config-manager --add-repo https://packages.centreon.com/rpm-standard/$MAJOR_VERSION/el9/centreon-$MAJOR_VERSION.repo
+              dnf config-manager --set-enabled centreon-$MAJOR_VERSION-stable*
+              dnf module reset -y php
+              dnf module install -y php:$(cut -d '-' -f 2 <<< ${PHP_REPO})
+              dnf install -y --allowerasing sudo curl
+              curl -LsS https://r.mariadb.com/downloads/mariadb_repo_setup \
+                | sudo bash -s -- --os-type=rhel --os-version=9 --mariadb-server-version="mariadb-10.5"
+              dnf install -y mariadb-server mariadb
+
+              if [[ "${{ inputs.module }}" == "web" ]];then
+                dnf download --resolve -y centreon-${{ inputs.module }}
+              else
+                dnf download --resolve -y centreon
+                dnf download --resolve -y centreon-${{ inputs.module }}
+              fi
+            ;;
+            debian:bullseye:23.10 | debian:bullseye:23.04 | debian:bullseye:22.10 | debian:bullseye:22.04)
+              apt update
+              apt update && apt install -y lsb-release ca-certificates apt-transport-https software-properties-common wget gnupg2 curl sudo
+              echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/sury-php.list
+              wget -O- https://packages.sury.org/php/apt.gpg | gpg --dearmor | tee /etc/apt/trusted.gpg.d/php.gpg  > /dev/null 2>&1
+              apt update
+              curl -LsS https://r.mariadb.com/downloads/mariadb_repo_setup | sudo bash -s -- --os-type=debian --os-version=11 --mariadb-server-version="mariadb-10.5"
+              echo "deb https://packages.centreon.com/apt-standard-$MAJOR_VERSION-stable/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/centreon.list
+              echo "deb https://packages.centreon.com/apt-plugins-stable/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/centreon-plugins.list
+              wget -O- https://apt-key.centreon.com | gpg --dearmor | tee /etc/apt/trusted.gpg.d/centreon.gpg > /dev/null 2>&1
+              apt update
+
+              if [[ "${{ inputs.module }}" == "web" ]];then
+                apt-get install --download-only -y centreon-${{ inputs.module }}
+              else
+                apt-get install --download-only -y centreon
+                apt-get install --download-only -y centreon-${{ inputs.module }}
```

Comment:
as we deliver only centreon's package, why noto wget it directly, instead of deploying the whole platform ?
Wasn't this step intended to check ".. that  packages delivered are reachable by the package manager client"

Reply:
it will download anything that is needed for centreon-${{ inputs.module }} wether i put it in a separate command or in a single one. it s just here for clarity.